gpflow/likelihoods/multilatent.py

```diff
@@ -120,3 +120,9 @@ def conditional_distribution(Fs) -> tfp.distributions.Distribution:
         super().__init__(
             latent_dim=2, conditional_distribution=conditional_distribution, **kwargs,
         )
+
+    def conditional_parameters(self, F):
```

Comment:
Because this is quite an important method, would it be possible to add docstrings?